src/EnergyPlus/SingleDuct.hh

```diff
@@ -122,9 +118,29 @@ namespace SingleDuct {
 
     // Types
 
+    struct SingleDuctAirTerminalFlowConditions
+    {
+        // Members
+        Real64 AirMassFlowRate;         // MassFlow through the Sys being Simulated [kg/Sec]
+        Real64 AirMassFlowRateMaxAvail; // MassFlow through the Sys being Simulated [kg/Sec]
+        Real64 AirMassFlowRateMinAvail; // MassFlow through the Sys being Simulated [kg/Sec]
+        Real64 AirTemp;                 // (C)
+        Real64 AirHumRat;               // (Kg/Kg)
+        Real64 AirEnthalpy;             // (J/Kg)
+        Real64 AirPressure;
+
+        // Default Constructor
+        SingleDuctAirTerminalFlowConditions()
+            : AirMassFlowRate(0.0), AirMassFlowRateMaxAvail(0.0), AirMassFlowRateMinAvail(0.0), AirTemp(0.0), AirHumRat(0.0), AirEnthalpy(0.0),
+            AirPressure(0.0)
+        {
+        }
+    };
+
     struct SingleDuctAirTerminal
     {
         // Members
+        int SysNum;               // index to single duct air terminal unit
```

Comment:
added to elimnate the need passing air terminal index as an argument of member function